docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,116 @@
+
+The iGPU 780M of AMD Ryzen CPU could run on ROCm at Linux with extral settings. 
+
+Available CPU: Ryzen 7000s/8000s CPU with iGPU 780M
```

Comment:
It might be helpful to summarize the key points here, and then break them down in detail below.  I believe the key points are:

- Ryzen 7000s/8000s CPU with iGPU 780M 
- amdgpu driver is required (is it?)
- Linux OS is required (Windows and WSL2 are not supported)
- BIOS must be set to enable the iGPU and dedicate > 1G RAM to VRAM
- HSA_OVERRIDE_GFX_VERSION="11.0.0" is set

Reply:
Good point.